media/juxtapose/css/playhead.css

```diff
@@ -4,6 +4,9 @@
 .jux-playhead-container input[type=range] {
     cursor: col-resize;
     width: 930px;
+    -webkit-appearance: none;
```

Comment:
drat. I was basing it off this: https://css-tricks.com/styling-cross-browser-compatible-range-inputs-css/.